Lib/cu2qu/rf.py

```diff
@@ -46,6 +44,103 @@ def zip(*args):
     return _zip(*args)
 
 
+class GetSegmentsPen(AbstractPen):
```

Comment:
Right.  I think we should just verify that the segments are compatible and pass them on unmodified.